### DISCUSSION TITLE
v1/zh-hans/docs/widgets/reward/

### DISCUSSION THREAD

**Discussion by giscus** [razonyang/hugo-theme-bootstrap#965] [General]
# v1/zh-hans/docs/widgets/reward/

打赏小部件，又称 buy me a coffee 小部件，其通过使用捐赠或小费，为读者提供了一种支持创作者的管道。
然而其只能用于二维码收款，其他请移步 funding widget。

https://hbs.razonyang.com/v1/zh-hans/docs/widgets/reward/

**Comment by phenix3443** [comment]
funding 和 reward 两个小部件的区别是什么？

**Reply by razonyang** [reply]
reward 只能显示二维码图片，funding 则是文本链接。
